haystack/components/evaluators/context_relevance.py

```diff
@@ -173,24 +182,23 @@ def run(self, questions: List[str], contexts: List[List[str]]) -> Dict[str, Any]
         :returns:
             A dictionary with the following outputs:
                 - `score`: Mean context relevance score over all the provided input questions.
-                - `individual_scores`: A list of context relevance scores for each input question.
-                - `results`: A list of dictionaries with `statements` and `statement_scores` for each input context.
+                - `results`: A list of dictionaries with `relevant_statements` and `score` for each input context.
         """
         result = super(ContextRelevanceEvaluator, self).run(questions=questions, contexts=contexts)
 
-        # calculate average statement relevance score per query
         for idx, res in enumerate(result["results"]):
             if res is None:
-                result["results"][idx] = {"statements": [], "statement_scores": [], "score": float("nan")}
+                result["results"][idx] = {"relevant_statements": [], "score": float("nan")}
                 continue
-            if not res["statements"]:
-                res["score"] = 0
-            else:
-                res["score"] = np_mean(res["statement_scores"])
+            if "relevant_statements" in res:
```

Comment:
Do we need this check? If we don't raise on failure,  `LLMEvaluator.run` would return a `None`, which we handle in line 191.